nilearn/_utils/docs.py

```diff
@@ -882,6 +882,32 @@ def custom_function(vertices):
     This parameter is passed to :func:`nilearn.signal.clean`.
 """
 
+docdict["screening_percentile"] = """
+screening_percentile : int, float, \
+                       in the closed interval [0, 100], or None, \
+                       default=20
```

Comment:
note that some function that use this doc string do not have this default but they are all private functions, so I feel the default=20 can stay there anyway.